ports/megamanx816-bit/port.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": 3,
+  "name": "megamanx816-bit.zip",
+  "items": [
+    "Mega Man X8 16-bit.sh",
+    "megamanx816-bit"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Mega Man X8 16-bit",
+    "porter": [
+      "Damian2809"
+    ],
+    "desc": "The Mega Man X8 16-Bit Demake is a fan-made project that reimagines the original Mega Man X8 in the style of the classic 16-bit era. This demake brings the modern aesthetics and mechanics of the original game into a retro environment reminiscent of the SNES era, particularly the earlier Mega Man X titles on that system.",
+    "desc_md": null,
+    "inst": "https://sonicfangameshq.com/forums/showcase/mega-man-x8-16-bit-demo3.1504/ grab the zip from here, extract it and rename the File to "Megaman X8 FC.pck" after that put it in megamanx816-bit/gamedata folder",
```

Comment:
Invalid JSON!
Error: Parse error on line 16:
...rename the File to "Megaman X8 FC.pck" a
-----------------------^
Expecting 'EOF', '}', ':', ',', ']', got 'undefined'

You have 3x " in there